source/index.slim

```diff
@@ -20,7 +20,15 @@ nav_logo_class: 'is-animated'
 
 - content_for :javascripts
   javascript:
-    $('#HomepageHero').load(function() {
-      var svg = $('#HomepageHero').contents().find('svg')[0];
-      window.homepage.animateLines(svg);
-    })
+    var heroWrapper = $('#HomepageHero')[0];
+    var objectElement = $('#HomepageHeroSVG');
+
+    function svgFinder() {
```

Comment:
is it just me, or is this method's name confusing?

without looking at it's content, I'm unable to tell whether `svgFinder()` would return the found svg, or a callback that would then find the svg

Not sure if I explained myself properly...